### PR TITLE
feat(styles): skeleton

### PR DESCRIPTION
**Pull request by platon-rov**
## Related Issue

Refers to https://github.com/SAP/fundamental-ngx/pull/8258
Refers to dxp/ngx-dxp/75

## Description

Skeleton styles & component.

### After:

image

### CODE REVIEW

**Review by g-cheishvili — APPROVED** [review]
Nice

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]
Blocked as it needs some work after today's discusssion.

File: stories/input-group/__snapshots__/input-group.stories.storyshot [hunk 1]
```diff
@@ -35,6 +35,7 @@ exports[`Storyshots Components/Forms/Input Group Focus 1`] = `
         class="fd-input fd-input-group__input"
```
Comment (mikerodonnell89) [inline stories/input-group/__snapshots__/input-group.stories.storyshot:35]: accidental commit?

File: src/styles/skeleton.scss [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  --fdSkeletonWidth: 100%;
+
+  position: relative;
+  color: transparent !important;
+  background-color: transparent !important;
+  box-shadow: none !important;
+  text-shadow: none !important;
+  border: none !important;
+  cursor: wait !important;
+  pointer-events: none !important;
```
Comment (mikerodonnell89) [inline src/styles/skeleton.scss:18]: `pointer-events: none` breaks `cursor: wait`

**Review by mikerodonnell89 — CHANGES_REQUESTED** [review]

File: src/styles/skeleton.scss [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  min-height: 1em !important;
+  max-height: 100% !important;
+  min-width: 1em !important;
+  max-width: 100% !important;
+  overflow: hidden !important;
+
+  /** Temporary */
+  &:not(.#{$block}--component) {
+    margin-top: 0.25rem !important;
+    margin-bottom: 0.25rem !important;
```
Comment (InnaAtanasova) [inline src/styles/skeleton.scss:27]: Please use mixin `fd-set-margins-y-equal`

File: src/styles/skeleton.scss [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+    .#{$block}:not(.#{$block}--component) {
+      margin-top: 0 !important;
+      margin-bottom: 0 !important;
```
Comment (InnaAtanasova) [inline src/styles/skeleton.scss:31]: Please use mixin `fd-set-margins-y-equal`

File: src/styles/skeleton.scss [hunk 5]
```diff
@@ -0,0 +1,76 @@
+@import './new-settings';
+@import './mixins';
+
+$block: fd-skeleton;
+
+.#{$block} {
+  @include fd-reset();
```
Comment (InnaAtanasova) [inline src/styles/skeleton.scss:7]: why do you need the reset?
I see cases like this: `fd-button fd-button--primary fd-skeleton`, which means that `fd-skeleton` can be/is a "modifier" for another component, and by adding the reset here you are changing some of the properties of the main component, such as margins, paddings, etc. 
Is this desired behavior?

**Pull request closed without merging**